Assets/_BForBoss/_Entities/Scripts/Navigation/Editor/AgentNavigationBehaviourEditor.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Perigon.Entities
```

Comment:
Can you move this to the _Entities/Scripts folder

Reply:
Updated